R/load_files.R

```diff
@@ -1078,17 +1078,17 @@ read_in_results_Tecan <- function(results_file, results_sheets, headers) {
     # rows and columns in data matrix with row and col names
     n_row <- nrow(dfm)
     n_col <- ncol(dfm)
-    readout <- as.data.frame(dfm[2:n_row, 2:n_col, with = FALSE])
-    rownames(readout) <- t(dfm[2:n_row, 1])
-    colnames(readout) <- dfm[1, 2:n_col]
+    readout <- dfm[2:n_row, 2:n_col, with = FALSE]
+    rownames(readout) <- as.character(t(dfm[2:n_row, 1]))
```

Comment:
`data.table` object does have rownames attribute so this line is obsolete

Reply:
Here is my suggestion to tackle it: 1b91dc92a7fb7cdf43ae9d28da057d9f93ca454f. What do you think?

Reply:
I mean we can get values of rownames using rownames(dt) but it isn't displayed in the object. So here if we want to assign it, rownames(dt) and row.names(dt) has the same functionality

Reply:
FYI: 
```
> dt <- data.table(lt = letters, LT = LETTERS)
> attributes(dt)
$names
[1] "lt" "LT"

$row.names
 [1]  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26

$class
[1] "data.table" "data.frame"

$.internal.selfref
<pointer: 0x55627cfffed0>
```

Reply:
Yes, but
```
> df <- data.frame(lt = letters, LT = LETTERS)
> rownames(df) <- LETTERS
> df["A", ]
  lt LT
A  a  A
```

but in data.table:

```
> dt <- data.table(lt = letters, LT = LETTERS)
> rownames(dt) <- LETTERS
> dt["A", ]
Error in `[.data.table`(dt, "A", ) : 
  When i is a data.table (or character vector), the columns to join by must be specified using 'on=' argument (see ?data.table), by keying x (i.e. sorted, and, marked as sorted, see ?setkey), or by sharing column names between x and i (i.e., a natural join). Keyed joins might have further speed benefits on very large data due to x being sorted in RAM.
```

Reply:
And after subsetting, rownames in dt are resetted while in df we still have there:
```
> rownames(dt[2, ])
[1] "1"
> rownames(df[2, ])
[1] "B"
```

Reply:
In the final version, I've removed the code for getting/setting rownames/colnames in data.table object.